ui/src/views/infra/zone/ZoneWizardAddResources.vue

```diff
@@ -846,9 +853,16 @@ export default {
       primaryStorageScopes: [],
       primaryStorageProtocols: [],
       primaryStorageProviders: [],
+      archTypes: [{
```

Comment:
there is an error with github action `Lint`
maybe use `architectureTypes` instead

Reply:
I think we could add it to the spelling file, `.github/linters/codespell.txt` as well. but changing the id should work as well.

Reply:
actually, a full name looks better to me.

Reply:
actually, a full name looks better to me. This might require adding to the codespell file as well, though.

Reply:
Thanks @weizhouapache @DaanHoogland - fixed